libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -205,6 +212,17 @@ export class ModelOverview extends React.Component<
       );
     }
 
+    if (this.state.datasetCohortLabeledStatistics.length === 0) {
```

Comment:
This makes sense. I do wonder what would happen if the request failed and we can't get the data. Will we just ask the user to wait forever?